arduino/main/lib/src/UltraSoundReader.cpp

```diff
@@ -3,30 +3,28 @@
 #include "Arduino.h"
 
 
-UltraSoundReader::UltraSoundReader(int trigger_pin, int echo_pin) {
+UltraSoundReader::UltraSoundReader(int trigger_pin, int echo_pin) : _trigger(trigger_pin), _echo(echo_pin){
 	pinMode(echo_pin, INPUT);
-    pinMode(trigger_pin, OUTPUT);
-	this->trigger = trigger_pin;
-	this->echo = echo_pin;
+  pinMode(trigger_pin, OUTPUT);
```

Comment:
Formatting seems to get messed up when I change it. I blame original tab indentation :D